dts/bindings/sensor/ti,fdc2x1x.yaml

```diff
@@ -0,0 +1,235 @@
+# Copyright (c) 2020 arithmetics ITS
+# SPDX-License-Identifier: Apache-2.0
+
+description: Texas Instruments FDC2X1X capacitive sensor
+
+compatible: "ti,fdc2x1x"
+
+include: i2c-device.yaml
+
+properties:
+    sd-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The SD pin defaults to active high when consumed by the sensor.
+        The property value should ensure the flags properly describe
+        the signal that is presented to the driver.
+
+    intb-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        The INTB pin defaults to active low when produced by the sensor.
+        The property value should ensure the flags properly describe
+        the signal that is presented to the driver.
+
+    fdc2x14:
+      type: boolean
+      description: |
+        Set to identify the sensor as FDC2114 or FDC2214 (4-channel version)
+
+    autoscan:
+      type: string
+      default: "enabled"
+      description: |
+        Set the Auto-Scan Mode.
+
+        disabled = Continuous conversion on the single channel selected by
+        active-channel.
+        enabled = Auto-Scan conversions as selected by rr-sequence.
+      enum:
+        - "disabled"
+        - "enabled"
+
+    rr-sequence:
+      type: int
+      required: false
+      default: 0
```

Comment:
Defaults in bindings must be justified; please explain why its value is set in the description.

If this value corresponds to the power-on-reset behavior of the device and the property can be changed independently of other properties, that's OK. If this is an individual preference made to work with the sample application or something like that, the property should not have the default value (and may need to be made `required` instead if there's no sane default corresponding to PoR behavior or something like that).

Reply:
Most of those values are power-on-reset defaults. The ones which are not I changed to required. I also added an explanation in each description for this and your comments below